ui/src/router/details/components/router_details_header/RouterDetailsPageHeader.js

```diff
@@ -24,7 +26,11 @@ export const RouterDetailsPageHeader = ({ router }) => {
     },
     {
       title: "Timeout",
-      description: <EuiText size="s">{router.config.timeout}</EuiText>,
+      description: (
+        <EuiTextColor size="s" color={isRouterUpdating ? "subdued" : "default"}>
```

Comment:
Using `EuiTextColor` directly instead of `<EuiText size="s"><EuiTextColor color="subdued">...` because with the latter, there is a vertical small alignment issue that I cannot easily identify the fix for - it seems the Timeout's value is placed a few pixels higher than all others in the header section. Eg:

![Screenshot 2021-11-01 at 9 28 32 AM](https://user-images.githubusercontent.com/23465343/139609794-16e9f981-9182-4704-a5e6-764ded01b381.png)